tests/test_routes.py

```diff
@@ -73,3 +76,40 @@ def test_index(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
     # Todo: Add your test cases here...
+    # ----------------------------------------------------------
+    # TEST CREATE
+    # ----------------------------------------------------------
+    def test_create_order(self):
+        """It should Create a new Order"""
+        test_order = OrderFactory()
+        logging.debug("Test Order: %s", test_order.serialize())
+        response = self.client.post(BASE_URL, json=test_order.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set
+        location = response.headers.get("Location", None)
+        self.assertIsNotNone(location)
+
+        # Check the data is correct
+        new_order = response.get_json()
+        # Avoid testing id, since it is generated by database
+        self.assertEqual(new_order["id"], test_order.id)
```

Comment:
I remember that you mentioned about the problem with testing id. Do we need to test it here or not?